script.js

```diff
@@ -158,12 +162,14 @@ const stockMathInfo = async (year_number, month_number, day_number, symbol) => {
         if(i == 0){
             startingStockValue = stockObject["Time Series (Daily)"][`${year_number}-${month_number}-${day_number}`]["1. open"]
             prices.push(parseInt(startingStockValue))
+            // COMMENT(andymina)
```

Comment:
I don't think these continue statements are working how you'd expect; using continue skips the iteration. For starters, this continue on L166 isn't needed because the loop doesn't run anything else so there's nothing else to skip. In the else statement below using continue probably won't get you all of the days you'd expect. Take this example:

```javascript
for (let i = 0; i < 5; i++) {
	if (i === 3) { continue; }
	console.log(`The number is ${i}`);
}
```

which would give 0 1 2 4 5, skipping 3 entirely.